### ISSUE TITLE
Trailing spaces following table header prevent single column tables from rendering

### ISSUE THREAD

**Issue by captainwillii** [showdownjs/showdown#442]
**Description:** If there is a trailing space following the header of a single column table, the table does not get rendered.  Multi column tables still render when there header has a trailing space.

**Input:**
```
| Single column | 
|:-------|
|   Row one   |
|   Row two   |
```
**Expected output:**

| Single column | 
|:-------|
|   Row one   |
|   Row two   |

**Actual Output:**
| Single column | |:-------| | Row one | | Row two |

**Comment by tivie** [comment]
Verified. The issue is not that serious and it should be an easy fix.

**Comment by tivie** [comment]
Should be fixed in v 1.7.6

---

>As you know, ShowdownJS is a free library and it will remain free forever. However, maintaining and improving the library costs time and money. Currently, we're looking to improve showdown with automated tests in all browsers and a proper domain and webpage. 500$ should be enough to to keep showdown testing framework running for a year or two.

**If you like our work and find our library useful, please donate [through Pledgie](https://pledgie.com/campaigns/34908) or directly [through paypal](https://www.paypal.me/tiviesantos)!! Your contribution will be greatly appreciated.**
